### DISCUSSION TITLE
Duda sobre Unidades del programa involucradas

### DISCUSSION THREAD

**Discussion by Darkonis13** [dds-jv/foro#314] [TPA - 2021] [answered]
Buenas profes!

Estaba repasando las clases y me pareció interesante  que en el enunciado del TPA4, se muestran las unidades involucradas.  ¿Dónde se puede ver donde empieza una unidad y donde termina? Es decir, ¿La unidad 5 sería la clase 5 por ejemplo?

Muchas gracias!

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Darkonis13 

No tiene relación con las clases en el sentido numerico. Distintas clases del cuatrimestre abarcan ciertos puntos del programa y algunos temas se van dando en distintas clases o incluso durante todo el año.

Por ejemplo la unidad 2, se va a trabajar todo el año porque se trata de como comunicar tu diseño en sus distintos aspectos.

Si queres saber que abarca el TPA4 sería todo desde que volvimos de vacaciones hasta la clase de HTTP.

**Reply by Darkonis13** [reply]
Muchas gracias profe! :D
